lte/gateway/python/magma/mobilityd/ipv6_allocator_pool.py

```diff
@@ -89,11 +89,11 @@ def remove_ip_blocks(self, ipblocks: List[ip_network],
         for sid in list(self._store.sid_ips_map):
             ip_desc = self._store.sid_ips_map[sid]
             if ip_desc.ip.version == 6:
```

Comment:
There is one more instance of ip_state_map, that needs to be changed  in this funciton.